### PR TITLE
make RPG Maker MV display separately

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
### SteamDB app page links to a few games using this
https://steamdb.info/app/1288310/
https://steamdb.info/app/1150690/

### Brief explanation of the change

added evidence for RPG Maker MV?

### CODE REVIEW

**Comment by diedyesterdaywashere** [comment]
yo @MichaelHinrichs i'm having the same issue with testing, my guess is that they didn't account for people making evidence stuff?

**Comment by diedyesterdaywashere** [comment]
i'll wait for one of the collaborators to figure it out because idk...

**Comment by xPaw** [comment]
Move Engine.RPG_Maker_MV into filelists folder, because there is no direct rule for it.

**Comment by MichaelHinrichs** [comment]
> And besides, what's the good reason to separate it out?

Because versions of RPG maker are massively different.

**Comment by diedyesterdaywashere** [comment]
> Move Engine.RPG_Maker_MV into filelists folder, because there is no direct rule for it.
> 
> And since you made it also match rpgmaker as a special rule, the filelist has to contain different file names for both to match. Is there are reason you added php code for this though?
> 
> And besides, what's the good reason to separate it out?

I added php code because that's how @MichaelHinrichs did it and it's the only way I seen.

RPG Makers are pretty different so it would be nice to differentiate them.

**Comment by diedyesterdaywashere** [comment]
removed all the evidence/php stuff

File: descriptions/Engine.RPG_Maker_MV.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**RPG MAKER MV**](https://www.rpgmakerweb.com/products/rpg-maker-mv) allows you to make the RPG of your dreams! This latest version includes a host of new featuresand export options for MacOSX, Android, and iPhone!
```
Comment (Lifeismana) [inline descriptions/Engine.RPG_Maker_MV.md:1]: You don't have to keep the engine's "marketing" and mention the platforms it supports. 
mentioning when it released and who made it is enough

**Comment by MichaelHinrichs** [comment]
> I added php code because that's how @MichaelHinrichs did it and it's the only way I seen.

Hey man, don't look at me. I barely understand what I'm doing with the Evidence files either.

**Comment by diedyesterdaywashere** [comment]
> > I added php code because that's how @MichaelHinrichs did it and it's the only way I seen.
> 
> Hey man, don't look at me. I barely understand what I'm doing with the Evidence files either.

I mean you were the only one whose pr had any evidence stuff in a while, but yeah, for now I'll just make a basic thing and then maybe someone who knows how to deal with it will make all the evidence stuff

**Comment by diedyesterdaywashere** [comment]
superseded by #393

**Pull request closed without merging**